instrumentation/opentelemetry-instrumentation-logging/src/opentelemetry/instrumentation/logging/__init__.py

```diff
@@ -46,6 +46,26 @@
 }
 
 
+def apply_log_settings(
+    format: str, level: int  # pylint: disable=redefined-builtin
+) -> None:
+    """
+    Apply the logging format to the root logger.
+
+    Ensures that the logging format is applied either by calling logging.basicConfig
+    or by setting the logging format on the root logger.
+    This is required in case of auto-instrumentation, where the root logger is configured
+    before we can inject the logging format here.
+    """
+    root_logger = logging.getLogger()
+    if root_logger.hasHandlers():
+        root_logger.setLevel(level)
+        for handler in root_logger.handlers:
+            handler.setFormatter(logging.Formatter(format))
```

Comment:
Shouldn't we be only reconfiguring the format for the OpenTelemetry `LoggingHandler` and not all of the user's handlers?